src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from '@cypress/react';
+import RulesTable from './RulesTable';
+import { Provider } from 'react-redux';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations/';
+import { getStore } from '../../Store';
+import fixtures from '../../../cypress/fixtures/rulesfixtures';
+import _ from 'lodash';
+
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
+import {
+  checkTableHeaders,
+  checkPaginationTotal,
+  checkPaginationValues,
+  PAGINATION_VALUES,
+  TOOLBAR,
+  tableIsSortedBy,
+  hasChip,
+  CHIP_GROUP,
+  changePagination,
+  checkRowCounts,
+  itemsPerPage,
+  PAGINATION,
+  PAGINATION_MENU,
+  DROPDOWN_TOGGLE,
+  SORTING_ORDERS,
+  checkSorting,
+  removeAllChips,
+  applyFilters,
+  filter,
+  checkNoMatchingRecs,
+  TOOLBAR_FILTER,
+  TITLE,
+  TABLE,
+  checkFiltering,
+} from '@redhat-cloud-services/frontend-components-utilities';
+
+//I'm looking at the https://docs.cypress.io/guides/component-testing/custom-mount-react#React-Router
+
+import { createIntl, createIntlCache } from 'react-intl';
+import { cellWidth, sortable, fitContent } from '@patternfly/react-table';
+import messages from '../../Messages';
+//declaring intl to provie it to the columns
+const cache = createIntlCache();
+const intl = createIntl(
+  {
+    // eslint-disable-next-line no-console
+    onError: console.error,
+    locale: navigator.language.slice(0, 2),
+  },
+  cache
+);
+//function and filters config for testing filters
+function* cumulativeCombinations(arr, current = []) {
+  let i = 0;
+  while (i < arr.length) {
+    let next = current.concat(arr[i]);
+    yield next;
+    i++;
+    const remaining = arr.slice(i);
+    if (remaining.length) {
+      yield* cumulativeCombinations(remaining, next);
+    }
+  }
+}
+const TOTAL_RISK = { Low: 1, Moderate: 2, Important: 3, Critical: 4 };
+const IMPACT = { Low: 1, Medium: 2, High: 3, Critical: 4 };
+const LIKELIHOOD = { Low: 1, Medium: 2, High: 3, Critical: 4 };
+const CATEGORIES = {
```

Comment:
please also double check that ocp-advisor rule categories are identical to what in the rhel ones

Reply:
We can add that to the utility library with the next update and import them from there. But i'm not sure that it's the best solution :)